frontend/src/app/facilitySelect/useSelectedFacility.ts

```diff
@@ -10,7 +10,7 @@ export function useSelectedFacility() {
   const facilityId = getFacilityIdFromUrl(location);
 
   const selectedFacility = useSelector<RootState, Facility | undefined>(
-    (state) => state.facilities.find((f) => f.id === facilityId)
+    (state) => state.facilities.find((f: Facility) => f.id === facilityId)
```

Comment:
just out of curiosity was this related to any of the dependency changes?

Reply:
Yep! The redux version 5 upgrade converted their codebase into Typescript and after upgrading that dependency along with react-redux, the frontend build was failing because of this being an `any` type.

Also related to the upgrade, the code in the other test file was importing DeepPartial which is no longer exported by that dependency but I checked what the import was on the main branch and it's just that type definition so I copied and pasted it.

Reply:
ahhh! Thank you for the explanation! 😸